src/Microsoft.TestPlatform.ObjectModel/Constants.cs

```diff
@@ -101,7 +101,13 @@ public static class Constants
         /// The default execution thread apartment state.
         /// </summary>
         [CLSCompliant(false)]
+#if NET451
```

Comment:
Do we have any tests for this default behavior?

Reply:
Yes, STA thread acceptance tests cover this